package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "demoblaze.com",
+  "version": "1.0.0",
+  "description": "Cypress automation project for demoblaze.com",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint . --ext .ts"
```

Comment:
Please add run and open npm script to run tests. 
Also, each npm script must include ts compile checking and lint checking, then run the test.